src/components/MyPersona/MyPersona.tsx

```diff
@@ -111,20 +96,42 @@ export const MyPersona: FC = () => {
     }
   }, [data, defaultPersonaUuid]);
 
+  const renderCarousel = useCallback(
+    (userPersonas) => (
+      <StyledCarousel
+        ref={carousel}
+        afterChange={(currentIndex) => {
+          setCurrentSlide(currentIndex);
+        }}
+      >
+        {userPersonas.map((persona: gqlEntity) => (
+          <CaruouselItem key={persona.uuid}>
+            <Wrapper
+              onClick={() =>
+                history.push({
+                  pathname: `${APP_ROUTES.PERSONA_PREVIEW(persona.uuid)}`,
+                })
+              }
+            >
+              <PersonaCard card={persona.card} uuid={persona.uuid} isWithEdit={true} />
+              {persona.uuid === defaultPersonaUuid ? (
+                <DefaultBlock>DEFAULT</DefaultBlock>
+              ) : (
+                <StyledButton onClick={() => handleSetDefault(persona.uuid)}>SET AS DEFAULT</StyledButton>
+              )}
+            </Wrapper>
+          </CaruouselItem>
+        ))}
+      </StyledCarousel>
+    ),
+    [data?.userPersonas]
```

Comment:
doesn't use callback do just shallow comparison, so `data.userPersonas` is always different?

Reply:
in deed - this is always the same, because reference is untouched, so probably this will recomputed twice - when userpersonas is undefined, and when is ready.